styles.css

```diff
@@ -0,0 +1,170 @@
+body {
+  background: url(images/Headerbg.svg) no-repeat;
```

Comment:
- [ ] The background image is visible only in the first middle of the page, I kindly invite you to fix this. This improvement will make your project awesome! 🐱 
![image](https://user-images.githubusercontent.com/20563252/124860537-6c89bb00-df77-11eb-8c1e-b114b3c81eaf.png)